velox/dwio/parquet/writer/Writer.cpp

```diff
@@ -386,6 +386,19 @@ parquet::WriterOptions getParquetOptions(
   if (options.compressionKind.has_value()) {
     parquetOptions.compression = options.compressionKind.value();
   }
+
+  if (options.flushPolicyFactory) {
+    if (auto flushPolicy = options.flushPolicyFactory()) {
```

Comment:
Should we set the DefaultFlushPolicy in the options itself instead of repeating this for Parquet and Dwrf?

Reply:
@majetideepak Good suggestions. Updated.